test/differencer/rulesProcessor.test.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { NodeDiff } from "../../src/diffTool/jsonDiff";
+import { applyRules } from "../../src/diffTool/rulesProcessor";
+import * as chai from "chai";
+import path from "path";
+import fs from "fs-extra";
+import tmp from "tmp";
+import sinon from "sinon";
+import { ramlToolLogger } from "../../src";
+import chaiAsPromised from "chai-as-promised";
+
+/* eslint-disable @typescript-eslint/no-use-before-define*/
+
+const expect = chai.expect;
+chai.use(chaiAsPromised);
+
+const rulesPath = path.join(__dirname, "../../diffRules", "defaultRules.json");
```

Comment:
This root `rulesPath` is shadowed in some tests, but not all, so I think it's harder to tell at a glance what `rulesPath` is in individual tests. I think it would reduce confusion if either this variable were renamed (`defaultRulesPath` or `validRulesPath` or something) or the single-use variables were renamed.

Reply:
renamed to validRulesPath